notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>prefer to import packages explicitly, it helps people to track down where different methods live</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
sure